src/services/resource.js

```diff
@@ -626,6 +630,119 @@ module.exports = class resourceHelper {
 		}
 	}
 
+	/**
+	 * Resource Details
+	 * @method
+	 * @name getDetails
+	 * @returns {JSON} - details of resource
+	 */
+	static async getDetails(resourceId) {
+		try {
+			let result = {
+				organization: {},
+			}
+
+			const resource = await resourceQueries.findOne({
+				id: resourceId,
+			})
+
+			if (!resource?.id) {
+				return responses.failureResponse({
+					message: 'RESOURCE_NOT_FOUND',
+					statusCode: httpStatusCode.bad_request,
+					responseCode: 'CLIENT_ERROR',
+				})
+			}
+
+			if (resource.blob_path) {
+				const response = await filesService.fetchJsonFromCloud(resource.blob_path)
+				if (
+					response.statusCode === httpStatusCode.ok &&
+					response.result &&
+					Object.keys(response.result).length > 0
+				) {
+					//modify the response as label value pair
+					let resultData = response.result
+
+					//get all entity types with entities
+					let entityTypes = await entityModelMappingQuery.findEntityTypesAndEntities(
+						{
+							model: resource.type,
```

Comment:
@priyanka-TL Don't  we need for other models such as task and subtask?

Reply:
Not required as of now. task and subtask dont have entities

Reply:
@priyanka-TL As discussed please add for future use case.